app/controllers/concerns/authentication/controller.rb

```diff
@@ -21,153 +20,185 @@ module Controller
-          begin
-            user = User.find(session[:user_id])
-            if user.persistence_token == session[:persistence_token]
-              set_current_user(user)
-            end
-          rescue ActiveRecord::RecordNotFound
-            # No need to do anything if the record does not exist
+    def load_session_user
+      if session[:user_id] && session[:persistence_token] && !current_user?
+        begin
+          user = User.find(session[:user_id])
+          if user.persistence_token == session[:persistence_token]
+            set_current_user(user)
           end
+        rescue ActiveRecord::RecordNotFound
+          # No need to do anything if the record does not exist
         end
       end
+    end
 
-      def handle_temporary_ban
-        if current_user? && current_user.temporary_banned?
-          logger.info "Authentication failed for user:#{current_user.id} (#{current_user.username}) - temporary ban"
-          flash[:notice] = "You have been banned for #{distance_of_time_in_words(Time.now, current_user.banned_until)}!"
-          deauthenticate!
-        end
+    def handle_temporary_ban
+      if current_user? && current_user.temporary_banned?
+        logger.info(
+          "Authentication failed for user:#{current_user.id} " +
+            "(#{current_user.username}) - temporary ban"
+        )
+        flash[:notice] =
+          "You have been banned for " +
+          distance_of_time_in_words(Time.now, current_user.banned_until) +
+          "!"
+        deauthenticate!
       end
+    end
 
-      def handle_permanent_ban
-        if current_user && current_user.banned?
-          logger.info "Authentication failed for user:#{current_user.id} (#{current_user.username}) - permanent ban"
-          flash[:notice] = "You have been banned!"
-          deauthenticate!
-        end
+    def handle_permanent_ban
+      if current_user && current_user.banned?
+        logger.info(
+          "Authentication failed for user:#{current_user.id} " +
+            "(#{current_user.username}) - permanent ban"
+        )
+        flash[:notice] = "You have been banned!"
+        deauthenticate!
       end
+    end
 
-      def verify_activated_account
-        if current_user?
-          logger.info "Authenticated as user:#{current_user.id} (#{current_user.username})"
-        end
+    def verify_activated_account
+      if current_user?
+        logger.info(
+          "Authenticated as user:#{current_user.id} " +
+            "(#{current_user.username})"
+        )
       end
+    end
 
-      def deauthenticate!
-        set_current_user(nil)
-      end
+    def deauthenticate!
+      set_current_user(nil)
+    end
 
-      def cleanup_temporary_ban
-        if current_user? && current_user.banned_until? && !current_user.temporary_banned?
-          current_user.update_attributes(banned_until: nil)
-        end
+    def cleanup_temporary_ban
+      if current_user? &&
+         current_user.banned_until? &&
```

Comment:
Use 4 (not 3) spaces for indenting a condition in an `if` statement spanning multiple lines.